REFER expectations

sfgeorge opened the pull request:
- Add support for expecting and accepting a REFER message.

sfgeorge committed:
Add support for receiving REFERs

sfgeorge committed:
[DOC] Categorize supported commands

sfgeorge committed:
[DOC] Categorize transfer/REFER commands separately

benlangfeld commented:
@sfgeorge Do you have those examples? Might you be able to rebase this PR so we can get it in to the next version?

sfgeorge commented:
Eek, sorry for the delay.  Happy Holidays, Happy New Year and Happy Saint Patrick's Day!  :four_leaf_clover:  :smile: 

I can rebase this weekend if that works.

What's the best format for an example?  I can send you pcap if that would suffice.

benlangfeld commented:
A pcap would be fine :)

chewi committed:
Fix non-writing of PCAP file when media is empty

This was broken by 893865f3d65e2f89ac9127c8457452fc6f66f0da.

benlangfeld committed:
Merge branch 'release/0.7.2' into develop

bklang committed:
Happy New Year

bklang committed:
Install pcap spec dependency

And upgrade test suite to cover Ruby 2.2 & 2.3

bklang committed:
Drop support for Ruby < 2.3

bklang committed:
Upgrade rspec to for modernization

bklang committed:
PacketFu 1.1.12 introduces a breaking change, removing PacketFu::UDPPacket

bklang committed:
Merge pull request #90 from mojolingo/feature/travis_fixes

Install pcap spec dependency

bklang committed:
Now require Ruby 2.3

chewi committed:
Use transpec to convert to new RSpec expect syntax

One test was already failing because it was somehow half-way between
the two syntaxes!

chewi committed:
Allow SIPp to be launched without sudo

It is not always necessary and it looks like the next version may not
need it at all.

bklang committed:
Merge pull request #97 from yakara-ltd/feature/nosudo

Allow SIPp to be launched without sudo

aukeman committed:
set event end flag for dtmf digits

bklang committed:
Merge pull request #99 from aukeman/dtmf-end-flag

set event end flag for dtmf digits

bklang committed:
Update Nokogiri for security issue

bklang committed:
Merge branch 'master' into develop

bklang committed:
Merge pull request #100 from mojolingo/feature/deps_update

Nokogiri update

pk1057 commented:
Hello,
i'm interested to get this feature up and running.
Does an scripting example exist to save me some time during testing ?

sfgeorge commented:
Hi @pk1057 my foul, I will try to update this PR and share for your and everyone's benefit.  Thank you for the reminder! 🎗

pk1057 commented:
Hi @sfgeorge, these are great news and i'm eagerly looking forward.
If i can help don't hesitate to contact me.

chewi committed:
PacketFu 1.1.12 didn't really drop PacketFu::UDPPacket

Loading of the various classes was just a bit broken. 1.1.13 works
fine and it supports Ruby 2.4.

bklang committed:
Merge pull request #101 from yakara-ltd/packetfu

PacketFu 1.1.12 didn't really drop PacketFu::UDPPacket

chewi committed:
Merge pull request #83 from yakara-ltd/bugfix/nomedia

Fix non-writing of PCAP file when media is empty

aukeman committed:
generate more robust DTMF events

aukeman committed:
Merge branch 'develop' of github.com:VHTx/sippy_cup into robust-dtmf-packets

bklang committed:
Merge pull request #103 from VHTx/robust-dtmf-packets

Robust dtmf packets

sfgeorge committed:
[CI] Add CI builds against C-Ruby 2.4, 2.5 and 2.6

sfgeorge committed:
[CI] Add ruby-2.5.3

sfgeorge committed:
[CI] Use Xenial build and rubies that are on it

sfgeorge committed:
[SPEC] Make spec a bit more lax for ruby-2.5

bklang committed:
Merge pull request #104 from sfgeorge/patch-1

[CI] Add CI builds against C-Ruby 2.4, 2.5 and 2.6

bklang committed:
Bump Nokogiri dep to address security concern

bklang committed:
Merge pull request #105 from mojolingo/feature/nokogiri_upgrade

Bump Nokogiri dep to address security concern
